src/core/zowe/core_for_zowe_sdk/config_file.py

```diff
@@ -238,7 +238,11 @@ def load_profile_properties(self, profile_name: str) -> dict:
         from the profile dict and populate fields from the secure credentials storage
         """
         try:
-            props = self.profiles[profile_name]["properties"]
+            props = self.profiles
```

Comment:
While passing `lpar1.zosmf` the `load_profile_properties` is getting `my_base` as profile_name, but it works fine `zosmf` where profile_name is `zosmf`.

Reply:
hey @t1m0thyj did some changes. Can you test that?

Reply:
@t1m0thyj here is the [team config files](https://github.com/zowe/zowe-client-python-sdk/files/11011719/config.zip) that I have tested for